### PR TITLE
Feature/oe 308 closer profit

### PR DESCRIPTION
**Pull request by ron-junior-devsquad**
## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: tests/Feature/CustomerTest.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 class CustomerTest extends TestCase
 {
-    use RefreshDatabase;
+    use DatabaseTransactions;
```
Comment (devsquad-mathews-lima) [inline tests/Feature/CustomerTest.php:22]: this will be **DatabaseTransactions**, or you just forgot to put back **RefreshDatabase**?

**Review by devsquad-mathews-lima — APPROVED** [review]
